external-reports/src/main/resources/org/sonar/l10n/java/rules/pmd/rules.json

```diff
@@ -0,0 +1,2306 @@
+[
+  {
+    "key": "AbstractClassWithoutAbstractMethod",
+    "name": "AbstractClassWithoutAbstractMethod",
+    "type": "CODE_SMELL",
+    "severity": "MAJOR",
```

Comment:
you provide 2 severities: for rule and issue, you should either drop this one or ignore one from report

Reply:
Changed in b56ab44